### PR TITLE
ADS: DaxTextInput Hide Password Icon

### PR DESCRIPTION
**Pull request by malmstein**
Task/Issue URL: https://app.asana.com/0/488551667048375/1204215011461545

### Description
Update the "hide password" icon used in DaxTextInput for when it is handling passwords.

### UI changes
| After  | Before |
| ------ | ----- |
Screenshot_20230324_100449|Screenshot_20230324_095901|

### CODE REVIEW

**Comment by malmstein** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #3002** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/3002" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/duckduckgo/Android/3002?utm_source=stack-comment).

File: common-ui/src/main/res/drawable/eye_closed_24.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+  <path
+      android:pathData="m4.707,20.707 l16,-16a1,1 0,1 0,-1.414 -1.414l-16,16a1,1 0,1 0,1.414 1.414ZM14.3,4.257c0.703,0.16 0.887,1.02 0.378,1.53l-0.092,0.091a1.077,1.077 0,0 1,-0.974 0.278A8.254,8.254 0,0 0,4.068 12c0.22,0.777 0.551,1.508 0.975,2.173 0.273,0.43 0.248,1 -0.113,1.361l-0.015,0.015c-0.415,0.415 -1.101,0.387 -1.428,-0.101a10.202,10.202 0,0 1,-1.423 -3.205L2.003,12l0.06,-0.242C3.178,7.302 7.207,4 12.009,4c0.788,0 1.555,0.089 2.292,0.257Z"
+      android:fillColor="#000"
+      android:fillAlpha="0.84"/>
+  <path
+      android:pathData="M12,8c0.149,0 0.296,0.008 0.44,0.024l-4.416,4.417A4,4 0,0 1,12 8ZM11.56,15.976l4.416,-4.417a4,4 0,0 1,-4.417 4.417Z"
+      android:fillColor="#000"
+      android:fillAlpha="0.84"/>
```
Comment (nalcalag) [inline common-ui/src/main/res/drawable/eye_closed_24.xml:13]: ditto
```suggestion
      android:fillColor="?attr/daxColorPrimaryIcon"/>
```

File: common-ui/src/main/res/drawable/eye_closed_24.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  <path
+      android:pathData="M10.392,17.842a1.077,1.077 0,0 0,-0.976 0.277l-0.09,0.091c-0.51,0.51 -0.326,1.368 0.376,1.53 0.741,0.17 1.513,0.26 2.306,0.26 4.803,0 8.831,-3.302 9.945,-7.758l0.06,-0.242 -0.06,-0.243a10.203,10.203 0,0 0,-1.43 -3.216c-0.328,-0.487 -1.014,-0.515 -1.429,-0.1l-0.014,0.015c-0.361,0.36 -0.386,0.932 -0.112,1.362 0.426,0.668 0.76,1.401 0.98,2.182a8.254,8.254 0,0 1,-9.556 5.842Z"
+      android:fillColor="#000"
+      android:fillAlpha="0.84"/>
```
Comment (nalcalag) [inline common-ui/src/main/res/drawable/eye_closed_24.xml:17]: Same 👍 
```suggestion
      android:fillColor="?attr/daxColorPrimaryIcon"/>
```

File: common-ui/src/main/res/drawable/eye_closed_24.xml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+  <path
+      android:pathData="m4.707,20.707 l16,-16a1,1 0,1 0,-1.414 -1.414l-16,16a1,1 0,1 0,1.414 1.414ZM14.3,4.257c0.703,0.16 0.887,1.02 0.378,1.53l-0.092,0.091a1.077,1.077 0,0 1,-0.974 0.278A8.254,8.254 0,0 0,4.068 12c0.22,0.777 0.551,1.508 0.975,2.173 0.273,0.43 0.248,1 -0.113,1.361l-0.015,0.015c-0.415,0.415 -1.101,0.387 -1.428,-0.101a10.202,10.202 0,0 1,-1.423 -3.205L2.003,12l0.06,-0.242C3.178,7.302 7.207,4 12.009,4c0.788,0 1.555,0.089 2.292,0.257Z"
+      android:fillColor="#000"
+      android:fillAlpha="0.84"/>
```
[the remark this answers is not in the document]
Comment (malmstein) [reply common-ui/src/main/res/drawable/eye_closed_24.xml]: we don't need this icon, it was just a copy

**Review by nalcalag — APPROVED** [review]
LGTM!

**Pull request merged**